addons/website_forum/models/forum.py

```diff
@@ -975,9 +971,13 @@ def _compute_website_url(self):
     def go_to_website(self):
         self.ensure_one()
         return {
-            'type': 'ir.actions.act_url',
-            'target': 'self',
-            'url': self.env['website'].get_client_action_url(self.website_url),
+            'type': 'ir.actions.client',
+            'tag': 'website_preview',
+            'context': {
+                'params': {
+                    'path': self.website_url,
+                }
+            }
```

Comment:
Why would not this use the new method? :thinking: I guess I'll test and change...